ENTACT_AGILENT/MSBNK-EPA-ENTACT_AGILENT001211.txt

```diff
@@ -1,26 +1,26 @@
 ACCESSION: MSBNK-EPA-ENTACT_AGILENT001211
-RECORD_TITLE: FD and C Green No. 1; ESI-QTOF; MS2; CE: 20; [M-H]-
+RECORD_TITLE: FD and C Green No. 1; ESI-QTOF; MS2; CE: 20; [M-2H]-
 DATE: 2023.09.19
 AUTHORS: EPA CCTE and Agilent Technologies
 LICENSE: CC BY
 CH$NAME: FD and C Green No. 1
-CH$NAME: DTXSID3020671
+CH$NAME: DTXSID5048001
 CH$COMPOUND_CLASS: N/A
-CH$FORMULA: [C37H37N2O6S2]+
-CH$EXACT_MASS: 669.2093033889
-CH$SMILES: CC[N+](CC1=CC(=CC=C1)S(O)(=O)=O)=C1C=CC(C=C1)=C(C1C=CC=CC=1)C1C=CC(=CC=1)N(CC1C=C(C=CC=1)S(O)(=O)=O)CC |c:14,t:21|
-CH$IUPAC: InChI=1S/C37H36N2O6S2/c1-3-38(26-28-10-8-14-35(24-28)46(40,41)42)33-20-16-31(17-21-33)37(30-12-6-5-7-13-30)32-18-22-34(23-19-32)39(4-2)27-29-11-9-15-36(25-29)47(43,44)45/h5-25H,3-4,26-27H2,1-2H3,(H-,40,41,42,43,44,45)/p+1
-CH$LINK: CAS 4680-78-8
-CH$LINK: INCHIKEY SRRJCDUOSQWHGS-UHFFFAOYSA-O
-CH$LINK: PUBCHEM CID:20803
+CH$FORMULA: C37H36N2O6S2
+CH$EXACT_MASS: 668.201479
+CH$SMILES: CCN(CC1=CC(=CC=C1)S(O)(=O)=O)C1=CC=C(C=C1)C(C1=CC=CC=C1)=C1C=CC(C=C1)=[N+](CC)CC1=CC(=CC=C1)S([O-])(=O)=O
+CH$IUPAC: InChI=1S/C37H36N2O6S2/c1-3-38(26-28-10-8-14-35(24-28)46(40,41)42)33-20-16-31(17-21-33)37(30-12-6-5-7-13-30)32-18-22-34(23-19-32)39(4-2)27-29-11-9-15-36(25-29)47(43,44)45/h5-25H,3-4,26-27H2,1-2H3,(H-,40,41,42,43,44,45)
+CH$LINK: CAS 6638-02-4
+CH$LINK: INCHIKEY SRRJCDUOSQWHGS-UHFFFAOYSA-N
+CH$LINK: PUBCHEM CID:73226
 AC$INSTRUMENT: N/A
 AC$INSTRUMENT_TYPE: ESI-QTOF
 AC$MASS_SPECTROMETRY: MS_TYPE MS2
 AC$MASS_SPECTROMETRY: ION_MODE NEGATIVE
 AC$MASS_SPECTROMETRY: IONIZATION ESI
 AC$MASS_SPECTROMETRY: COLLISION_ENERGY 20
 MS$FOCUSED_ION: PRECURSOR_M/Z 668.2020269372
-MS$FOCUSED_ION: PRECURSOR_TYPE [M-H]-
+MS$FOCUSED_ION: PRECURSOR_TYPE [M-2H]-
```

Comment:
If we keep this updated chemical information then this should be [M]- ... but the mass accuracy is not good (the precursor is higher mass than the exact mass, it should be slightly lower if it loses an electron) and the peak intensities in this record are extremely low.